server/src/main/scala/net/wiringbits/config/NotificationsConfig.scala

```diff
@@ -0,0 +1,20 @@
+package net.wiringbits.config
+
+import play.api.Configuration
+
+import scala.concurrent.duration.FiniteDuration
+
+case class NotificationsConfig(delayedInit: FiniteDuration, interval: FiniteDuration, delayIfFailure: FiniteDuration) {
+  override def toString: String = {
+    s"NotificationsConfig(delayedInit = $delayedInit, interval = $interval, delayIfFailure = $delayIfFailure)"
```

Comment:
`delayIfFailure` is not needed, let's use an exponential backoff strategy starting from 1 minute: 1m, 2m, 4m, 8m, 16m, etc.